pagopa/openapi.yaml

```diff
@@ -0,0 +1,89 @@
+openapi: 3.0.3
+info:
+  title: pagoPa RTP - OpenAPI 3.1
+  description: DESCRIPTION
+  version: 1.0.11
+servers:
+  - url: http://localhost:8080
+tags:
+  - name: rtp-notice
+    description: API Group aims to notify a pagoPa notice through Request To Pay (RTP)
+paths:
+  /rtp:
+    post:
+      tags:
+        - rtp-notice
+      summary: "Create and send a new RTP from a pagoPa notice"
+      description: ""
+      requestBody:
+        description: ""
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateRequestToPayNotice"
+      responses:
+        201:
```

Comment:
We have to define the behavior of the system: if the sending is asynchronous and this represents a takeover, it is more correct a 202 (accepted)